src/Store/ActionTypes.js

```diff
@@ -19,7 +19,7 @@ export const FILTER_PLATFORM_ITEMS = 'FILTER_PLATFORM_ITEMS';
 export const FETCH_PORTFOLIOS = 'FETCH_PORTFOLIOS';
 export const FETCH_PORTFOLIO_ITEMS = 'FETCH_PORTFOLIO_ITEMS';
 export const FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO = 'FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO';
-export const FETCH_PORTFOLIO_ITEM = 'FETCH_PORTFOLIO_ITEM';
+export const FETCH_PORTFOLIO = 'FETCH_PORTFOLIO';
```

Comment:
It's better to divide action types into more files, not to have them in only one file like this. The same as we have actions in `AlertActions.js`,`OrderActions.js`, `PlatformActions.js`,... We don't have them all in something like `Actions.js` (and the same for reducers) :)

Reply:
We only have a few at the moment,. Agree, as we have more actions, we should divide the file into store categories.